### ISSUE TITLE
Add vue warning in the checker

### ISSUE THREAD

**Issue by louia** [fi3ework/vite-plugin-checker#177]
### Is your feature request related to a problem? Please describe.

Currently, I cannot show vue warnings in the panel of this plugin. My vue warning only appear in the dev tools console of my browser.

### Describe the solution you'd like

Add vue warnings in consideration of this plugin

### Validations

- [X] Read the [docs](https://github.com/fi3ework/vite-plugin-checker#readme).
- [X] Read the [Contributing Guidelines](https://github.com/fi3ework/vite-plugin-checker/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that asks for the same feature to avoid creating a duplicate.

**Comment by fi3ework** [comment]
which vue checker are you using? `vue-tsc` or `vetur`?

**Comment by fi3ework** [comment]
Also, a minimal re-production will help a lot to fix the potential bug.

**Comment by Anubarak** [comment]
I made a minimal production repo here

https://github.com/Anubarak/test-vite-plugin-checker-flat-config

my code is
```
<script setup
        lang="ts"
>

const code = '{{ a }}';
const i = 0;
</script>

<template>
    html code is <code>{{ code }}</code> this should show a warning but it does not
    {{ a }}
</template>
```
that should produce 2 errors, one for missing variable (warning), one for unused variable (error) but only the error is shown. When you remove `const i = 0` the checker is empty

![image](https://github.com/user-attachments/assets/4fcc5d35-b5c3-45e9-a0a5-3b3db994d83a)

**Comment by danielroe** [comment]
You need to enable the `vue-tsc` integration by passing `vueTsc: true` to your checker options.

**Comment by yusufkandemir** [comment]
@danielroe there are still cases where a Vue runtime warning occurs, without vue-tsc catching a type error. This also applies when not using TS at all. So, this feature request is basically about making Vue runtime warnings in the developer console more visible by showing them in the overlay.
